Aztec/Classes/Libxml2/DOMString.swift

```diff
@@ -341,6 +341,11 @@ extension Libxml2 {
         
         // Apply Styles
         
+        fileprivate func apply(element: StandardElementType, spanning range: NSRange, attributes: [Attribute] = []) {
+            performAsyncUndoable { [weak self] in
```

Comment:
The idea here is to avoid repetitition of the performaAsyncUndoable everywhere. If in the future we change the way we do this, we can change in a single place.